pyproject.toml

```diff
@@ -143,6 +131,9 @@ strict = true
 disallow_any_unimported = true
 no_implicit_optional = true
 show_error_codes = true
+explicit_package_bases = true
```

Comment:
In workflow-common.yml you removed "explicit-package-bases" and here you activate it. Shouldn't it be false?

Reply:
I moved the configuration to pyproject.toml so pre-commit hook can use the same configuration. In other words, it's no longer in the workflow because it's defined here.

Reply:
ok